gadget/install/install.go

```diff
@@ -166,11 +171,17 @@ func Run(model gadget.Model, gadgetRoot, kernelRoot, device string, options Opti
 		// matches what is on the disk, but sometimes there may not be a sector
 		// size specified in the gadget.yaml, but we will always have the sector
 		// size from the physical disk device
-		if err := makeFilesystem(&part, diskLayout.SectorSize); err != nil {
+		sp = perfTimings.StartSpan("make-filesystem", fmt.Sprintf("Create filesystem for %s", part.Node))
+		err = makeFilesystem(&part, diskLayout.SectorSize)
+		sp.Stop()
```

Comment:
Nitpick^2, you could use
```
timings.Run(perfTimings, ... func(nesttm timings.Measurer) {
  err = makeFilesystem(&part, diskLayout.SectorSize)
})
```
to avoid explicit StartSpan/StopSpan here.

Reply:
I agree with stolowski about this

Reply:
typically when we do timings for things looped over we try to give unique labels:  we can probably use part.Role or part.Name if Role is empty

and labels of the form  "make-filesystem[role or name]"